src/github/client/github-client.ts

```diff
@@ -124,14 +107,28 @@ export default class GitHubClient {
 	}
 
 	private async graphql<T>(query: string, variables?: Record<string, string | number | undefined>): Promise<AxiosResponse<GraphQlQueryResponse<T>>> {
-		return  await this.axios.post<GraphQlQueryResponse<T>>("https://api.github.com/graphql",
+		const response = await this.axios.post<GraphQlQueryResponse<T>>("https://api.github.com/graphql",
```

Comment:
Not really due to this change, but: 

This url shouldn't be hard-coded, but instead something like`this.installationId.githubBaseUrl + "/graphql"` (or create a field `githubGraphqlBaseUrl` in the `InstallationId`.

Otherwise, we can't disambiguate between GHE and GH Cloud lateron.